cluster/src/main/java/io/scalecube/cluster/ClusterImpl.java

```diff
@@ -351,4 +352,66 @@ public NetworkEmulator networkEmulator() {
   public boolean isShutdown() {
     return onShutdown.isDisposed();
   }
+
+  public interface ClusterMonitorMBean {
```

Comment:
[Split this class into two classes](https://github.com/scalecube/scalecube-cluster/pull/174/files#r261251876)